front/src/app/todo-list/todo-list.component.scss

```diff
@@ -152,10 +152,14 @@
   }
 
   &__icon {
-    margin-right: 5px;
+    position: absolute;
+    right: 1px;

```

Comment:
there should be no need in `position-absolute` - take a look on the code you had earlier